pkg/util/admission/granter.go

```diff
@@ -1976,12 +2229,14 @@ func (res adjustTokensResult) SafeFormat(p redact.SafePrinter, _ rune) {
 	p.Printf("%s unacc [≈%s/req, n=%d], ",
 		ib(res.aux.intUnaccountedL0Bytes), ib(int64(res.smoothedIntPerWorkUnaccountedL0Bytes)), res.aux.intAdmittedCount)
 	// How much got compacted out of L0 recently.
-	p.Printf("compacted %s [≈%s]; ", ib(res.aux.intL0CompactedBytes), ib(res.smoothedIntL0CompactedBytes))
-
+	p.Printf("compacted %s [≈%s], ", ib(res.aux.intL0CompactedBytes), ib(res.smoothedIntL0CompactedBytes))
+	p.Printf("flush tokens %s with multiplier %.3f [≈%s]; ", ib(int64(res.aux.intFlushTokens)),
+		res.flushUtilTargetFraction, ib(int64(res.smoothedNumFlushTokens*res.flushUtilTargetFraction)))
 	p.Printf("admitting ")
 	if n := res.ioLoadListenerState.totalNumByteTokens; n < unlimitedTokens {
 		p.Printf("%s", ib(n))
```

Comment:
while we're here, can we make this a rate? So something like `"%s/s", ib(float64(n)/float64(intervalLength*time.Second))`?